README

```diff
@@ -54,6 +54,8 @@ Optional arguments
 --parallel-build      - allow parallel build of domains.
                         This is useful for building on powerful machines or
                         build servers.
+--prod_pulls	      - allow define the list of the pull requests applied to 
```

Comment:
May be it's better to use white spaces instead of tabs.
And for next line also.

Reply:
fixed